### DISCUSSION TITLE
Need to press any key twice

### DISCUSSION THREAD

**Discussion by rng70-or** [zed-industries/zed#17116] [Feature Requests]
I need to type any key twice to do something in zed while using shortcuts! for example if I need to paste something from clipboard what I need to do is Ctrl + V then again need to press V so (Ctrl + V + V). Am I missing something in the settings which is set by default? 
 
 Base keymap is set to 'VSCode'
 Platform: Ubuntu 23.10

**Comment by Moshyfawn** [comment]
This issue sounds similar to #16904

**Comment by rng70-or** [comment]
solved
